tests/framework/extensions/clusters/versions/default.go

```diff
@@ -0,0 +1,70 @@
+package versions
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/stretchr/testify/require"
+)
+
+func DefaultKubernetesVersion(t *testing.T, client *rancher.Client, provider string, kubernetesVersions []string) ([]string, error) {
+
+	switch {
+	case provider == "rke1":
```

Comment:
Since we use the same string here and in the tests, let's make them immutable strings. Check this for an example: 
https://github.com/rancher/rancher/blob/release/v2.7/tests/v2/validation/provisioning/config.go
we can put them here, and have the tests import them. Just to make things a tad bit cleaner.

Reply:
oooo.. okay! cool